setup.py

```diff
@@ -41,6 +78,32 @@
 if 'all' in sys.warnoptions:
     log.level = logging.DEBUG
 
+
+class GEOSConfig(object):
+    """Interface to config options from the `geos-config` utility
+    """
+
+    def __init__(self, cmd):
+        self.cmd = cmd
+
+    def get(self, option):
+        try:
+            output = subprocess.check_output([self.cmd, option])
```

Comment:
[`check_output`](https://docs.python.org/2/library/subprocess.html#subprocess.check_output) was annoyingly added in Python 2.7